Admin/Extension/MenuNodeReferrersExtension.php

```diff
@@ -21,20 +21,47 @@
  */
 class MenuNodeReferrersExtension extends AdminExtension
 {
+    /**
+     * @var string
+     */
+    private $formTab;
+
+    /**
+     * @var string
+     */
+    private $formGroup;
+
+    public function __construct($formGroup = 'form.group_menu', $formTab = 'form.tab_menu')
+    {
+        $this->formGroup = $formGroup;
+        $this->formTab = $formTab;
+    }
+
     public function configureFormFields(FormMapper $formMapper)
     {
+        if ($formMapper->hasOpenTab()) {
+            $formMapper->end();
+        }
```

Comment:
oh crap, only saw this comment now. i think it would be really good to bring back the tabs. the current admin looks rather bad with the huge lists of stuff instead of tabs.

shall we rewrite things to use method_exists('hasOpenTab') and swallow the exception if the method did not exist?

```
if (method_exists($formMapper, 'hasOpenTab')) {
        if ($formMapper->hasOpenTab()) {
            $formMapper->end();
       }
} else {
     try {
            $formMapper->end();
    } catch(TheException $e) {
         // ignore, there was no tab open.
    }
}
```